libcidr-shopify.rb

```diff
@@ -0,0 +1,12 @@
+class LibcidrShopify < Formula
+  desc 'Libcidr is a library that provides utility functions to handle IP addresses and net blocks.'
+  homepage 'https://github.com/Shopify/libcidr-shopify/'
+  url 'https://github.com/Shopify/libcidr-shopify/archive/v1.0.0.tar.gz'
+  sha256 '718fbb00ab1ae8f9e692ce6f917ee7b485420df1e05537f8ab871de6a667915d'
+
+  def install
+    system "make"
```

Comment:
does this project not require running `configure` ?

Reply:
are there source changes in libcidr-shopify from the upstream source? ie. do we need our own fork, or can we build from the upstream?

Reply:
a possible approach would be to download the authoritative tarball, extract the source, and apply a patch

Reply:
you can see we have done this sort of thing in other homebrew packages... https://github.com/Shopify/homebrew-shopify/blob/11b18d986cb4f30095faac03e65ec2cad13ef50a/nginx-shopify.rb#L27-L30

Reply:
wdyt @ElvinEfendi ?

Reply:
I like the idea of patching the upstream here in the formula as you suggest.

Reply:
@sbfaulkner The project's README does not mention running ``` ./configure ```. From the README: "To install from a release tarball, all you should need to do is a
  standard "make ; make install" procedure."

Reply:
@sbfaulkner There are source changes in libcidr-shopify from the upstream source. Mainly modifying some Makefiles to add support for macOS.

Reply:
@ElvinEfendi @sbfaulkner I agree, patching makes more sense in this case. I'm working on it.